inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/parser/ExistsParser.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.sdk.transform.process.parser;
+
+import org.apache.inlong.sdk.transform.decode.SourceData;
+import org.apache.inlong.sdk.transform.process.Context;
+import org.apache.inlong.sdk.transform.process.operator.OperatorTools;
+
+import lombok.extern.slf4j.Slf4j;
+import net.sf.jsqlparser.expression.Expression;
+import net.sf.jsqlparser.expression.operators.relational.ExistsExpression;
+/**
+ * ExistsParser
+ */
+@Slf4j
+@TransformParser(values = ExistsExpression.class)
+public class ExistsParser implements ValueParser {
+
+    private final ValueParser subQueryValueParser;
+
+    public ExistsParser(ExistsExpression expr) {
+        Expression rightExpression = expr.getRightExpression();
+        this.subQueryValueParser = OperatorTools.buildParser(rightExpression);
+    }
+
+    @Override
+    public Object parse(SourceData sourceData, int rowIndex, Context context) {
+        try {
```

Comment:
[NOT] EXISTS(subquery)
The semantic scenario of using EXISTS in subqueries, where the example is similar to the semantics of IS NULL.

Reply:
you're right, the current implementation is incorrect, Should the parsing of subqueries be at the function/parser/operator level or at the TransformProcessor level？it seems that the `initTransformSql` method in `TransformProcessor`  doesn't 
 handle the subquery